### ISSUE TITLE
"ERROR: libmp3lame >= 3.98.3 not found" when built with Windows

### ISSUE THREAD

**Issue by Coirt** [VCVRack/VCV-Recorder#17]
commit 6dd76d3

I have nasm installed as suggested in #15

```
cd dep/ffmpeg && ./configure --prefix="/home/Phil/rack-1.0.3/rack/plugins/vcv-recorder/dep" --enable-pic --enable-gpl \
        --disable-programs --disable-doc --disable-avdevice --disable-postproc --disable-avfilter --disable-network --disable-iconv --disable-alsa --disable-autodetect --disable-everything \
        --enable-protocol=file \
        --enable-muxer=wav --enable-encoder=pcm_s16le --enable-encoder=pcm_s24le \
        --enable-muxer=aiff --enable-encoder=pcm_s16be --enable-encoder=pcm_s24be \
        --enable-libmp3lame --enable-muxer=mp3 --enable-encoder=libmp3lame \
        --enable-muxer=flac --enable-encoder=flac \
        --enable-muxer=ipod --enable-encoder=alac \
        --enable-muxer=mpeg1system --enable-encoder=mpeg2video --enable-encoder=mp2
ERROR: libmp3lame >= 3.98.3 not found

If you think configure made a mistake, make sure you are using the latest
version from Git.  If the latest version fails, report the problem to the
ffmpeg-user@ffmpeg.org mailing list or IRC #ffmpeg on irc.freenode.net.
Include the log file "ffbuild/config.log" produced by configure as this will help
solve the problem.
make: *** [Makefile:32: dep/lib/libavcodec.a] Error 1

```

**Comment by MarcBoule** [comment]
Getting the same error.
Perhaps there are some useful answers here, but I'm not good enough with makefiles and build scripts to be of any help. 
https://stackoverflow.com/questions/35937403/error-libmp3lame-3-98-3-not-found

**Comment by SmallTownSound** [comment]
same error here

**Comment by Coirt** [comment]
To save anyone else trying the latest build, b2e56f6 is the same.

**Comment by stoermelder** [comment]
`pacman -Su mingw-w64-x86_64-lame`
fixed this error for me.

**Comment by MarcBoule** [comment]
@stoermelder Nice! That worked for me (Win10), now `make dep` completes without errors. Thanks for sharing :-)

**Comment by SmallTownSound** [comment]
Thanks...  Just tried pacman -Su mingw-w64-x86_64-lame, fixed make dep, but getting make error with latest pulls...  Will try later today to see if rack or recorder gets a new upload.

**Comment by Coirt** [comment]
One of the previous commits builds dep okay. 

For future reference with lame being installed as a package would that particular code export with `make dist` ?

**Comment by stoermelder** [comment]
Ok, installing the library is more a workaround than a real fix.
But adding following flags to `configure` of ffmpeg in the Makefile builds the dependencies as expected, without installing through pacman. I'm not so good with Makefiles though, so this is probably not how it should be done...
`--extra-cflags="$(FLAGS)" --extra-ldflags="-L../../dep/lib"`
#19 still persists.

**Comment by stoermelder** [comment]
Fixed in 267bec1cb97ae2ab7946286f25d73e9203ba206f
